src/ident/AclIdent.cc

```diff
@@ -56,10 +53,8 @@ int
 ACLIdent::match(ACLChecklist *cl)
 {
     const auto checklist = Filled(cl);
-    if (checklist->rfc931[0]) {
-        return data->match(checklist->rfc931);
-    } else if (checklist->conn() != nullptr && checklist->conn()->clientConnection != nullptr && checklist->conn()->clientConnection->rfc931[0]) {
-        return data->match(checklist->conn()->clientConnection->rfc931);
+    if (const auto ident = checklist->ident()) {
+        return data->match(ident.value().isEmpty() ? dash_str : SBuf(ident.value()).c_str());
```

Comment:
Format::assemble() tricks aside, this is the only case where PR treats an empty cached/stored ident value specially, right?

Reply:
Yes, those format cases are in log/Format*.cc.